recipes/purescript-mode

```diff
@@ -0,0 +1,8 @@
+(purescript-mode
+ :repo "dysinger/purescript-mode"
+ :fetcher github
+ :files ("*.el"
+         "purescript-mode.texi"
+         "NEWS"
```

Comment:
Is there a reason `NEWS` is included? I presume, a package should only include what's necessary for its usage. I doubt someone would be looking for NEWS file *(even if it's updated, [which it never was](https://github.com/purescript-emacs/purescript-mode/pull/55))* in the installation directory, I presume people would search for the repo instead. For one, because it's the only way you can read what's up in the project before you install an update.